### DISCUSSION TITLE
Lithium + Cocaine

### DISCUSSION THREAD

**Discussion by zelixir25** [TripSit/drugs#93] [Drug Interactions]
Still looking for research in humans but lithium seems to decrease some effects of cocaine in this study with a very small sample of rats. It at least seems safe, though!

Flemenbaum, A. (1977). Antagonism of behavioral effects of cocaine by lithium. Pharmacology Biochemistry and Behavior, 7(1), 83–85. https://doi.org/10.1016/0091-3057(77)90015-6

**Comment by utaninja** [comment]
Looks good to me, adding this source and pushing

  Cronson, A. J., & Flemenbaum, A. (1981). Antagonism of Cocaine Highs by Lithium. In A. J. Schecter (Ed.), Drug Dependence and Alcoholism: Volume 1 Biomedical Issues (pp. 1139–1141). Springer US. https://doi.org/10.1007/978-1-4684-3614-3_132

**Comment by utaninja** [comment]
Pushed to database https://github.com/TripSit/drugs/pull/129
